src/components/PostDetails.tsx

```diff
@@ -98,21 +75,21 @@
       </div>
 
       <div className="block">
-        {!loaded && <Loader />}
+        {loaded && <Loader />}
```

Comment:
The condition here should be `!loaded` to show the loader when the comments are being loaded. The current condition `loaded` will show the loader when loading is complete, which is incorrect.